extensions/wikia/WallNotifications/tests/WallNotificationEntityTest.php

```diff
@@ -463,7 +474,7 @@ public function loadDataFromRevDataProvider() {
 			],
 			'parentUser' => [
 			],
-			'entity' => [
+			'user' => [
 				'userName' => 'GarthTest400',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "GarthTest400" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)